Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -2389,7 +2391,7 @@ void DragCaretController::paintDragCaret(LocalFrame* frame, GraphicsContext& p,
 {
 #if ENABLE(TEXT_CARET)
     if (m_position.deepEquivalent().deprecatedNode() && m_position.deepEquivalent().deprecatedNode()->document().frame() == frame)
-        paintCaret(*m_position.deepEquivalent().deprecatedNode(), p, paintOffset, nullptr);
+        paintCaret(*m_position.deepEquivalent().deprecatedNode(), p, paintOffset, nullptr, false);
```

Comment:
Optional nit: boolean trap here. What about `enum class AppUsesCustomAccentColor : bool;` to improve readability at the call sites?